src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+    private final StringBuilder stringBuilder = new StringBuilder();
+
+    @Override
+    public String toString() {
+
+        return stringBuilder
+                .append("Ball's number: ").append(number)
+                .append("\nIt's color: ").append(color).toString();
+    }

```

Comment:
In this case, you can use regular String concatenation. And usually this method is placed at the end of class